modules/errorhandler-core/Debugger_Backtrace.asm

```diff
@@ -111,7 +111,7 @@ Debugger_Backtrace:	__global
 	@try_next_offset:
 		addq.l	#2, @stack_curr
 		cmpa.l	@stack_curr, @stack_top
-		bhs		@try_offset_loop
+		bhs.s		@try_offset_loop
```

Comment:
The only changes I'm going to exclude before merging are some Bxx -> Bxx.S optimizations, because some instances are redundant.

If you take a closer look at the listing file, ASM68K is able to optimize them perfectly fine:
```asm
00000EF6                            	@try_next_offset:
00000EF6 548A                       		addq.l	#2, @stack_curr
00000EF8 B3CA                       		cmpa.l	@stack_curr, @stack_top
00000EFA 6490                       		bhs		@try_offset_loop     ; 64 xx = BHS.S
```

ASM68K is able to do that because `@try_offset_loop` isn't a forward-reference, the label was defined above the `bhs` instruction and its offset is known at compile-time. If the label were below the instruction (forward-reference), that the optimization wouldn't be possible, because ASM68K is a 2-pass assembler (1st pass = compile code, 2nd pass = insert references).

As I rule of thumb, I never manually optimize backward branches, because:
  1. ASM68K can handle it,
  2. this is more flexible: I can add some heavy macro like `KDebug.WriteLine "Logging something..."` between the label and `bhs` and then I'd have to specify `bhs.w` in DEBUG builds and `bhs.s` in RELEASE builds (where KDebug is ignored).

Reply:
A fair point on those. I'm used to always specifying lengths for branches, so they were assembling as word-length in my code. I really should enable all optimizations in my build though.